src/post.ts

```diff
@@ -162,7 +162,7 @@ export default function postCombatActions(skipDiet = false): void {
   updateMallPrices();
   stillsuit();
   funguySpores();
-  if (globalOptions.ascend || AutumnAton.turnsForQuest() < myAdventures() + 10) {
-    AutumnAton.sendTo(autumnAtonZones);
+  if (globalOptions.ascend || AutumnAton.turnsForQuest() < estimatedTurns()) {
+    AutumnAton.sendTo((locations) => maxBy(mostValuableUpgrade(locations), averageAutumnatonValue));
   }
```

Comment:
I think instead of defining the thing in-line here using maxBy, mostValuableUpgrade, and averageAutumnAtonValue, `autumnaton.ts` should export a single function that either goes into the `AutumnAton.sendTo` or does the sendTo itself. That is, this should either end up looking like
`AutumnAton.sendTo(bestAutumnatonLocation)` or `sendFallbot()`, depending upon which of those feels more right

Reply:
I think a `bestAutumnatonLocation` feels better. Later if we decide it's worth the effort to figure out how to prepare and unlock zones we want to send autumnaton to, I think it will make more sense (And would probably change into two separate functions of `bestAvailableAutumnatonLocation` and `bestUnlockableAutumnatonLocation`